### PR TITLE
Update Signs and Markings custom code with up-to-date big button and SSO code

### PR DESCRIPTION
**Pull request by mddilley**
This PR fixes [the issue](https://github.com/cityofaustin/atd-data-tech/issues/4851) with the custom SSO buttons where the underline text decoration was not overridden and the custom SSO button and big buttons had margin to the right that was clickable. This was done by adding the updates from https://github.com/cityofaustin/atd-knack/pull/29. A big part of this PR is updating any instances of a legacy function called `customButton` with the latest big button code.

### Demo
signsAndMarkings

### CODE REVIEW

File: code/signs-markings/signs-markings.css [hunk 1]
```diff
@@ -70,6 +48,45 @@ a.small-button {
...
+  color: #07467c !important;
+  background: white !important;
+  border: 3px solid #07467c;
+  font-size: 22px;
+  font-weight: bold;
+  height: 44px;
+  padding: 0 15px;
+}
+
+//this prevents the natural underlining Knack does for links
```
Comment (chiaberry) [inline code/signs-markings/signs-markings.css:85]: I put a comment like this in the old knack builder css file and I thought it messed up / preventing the style from working. Does the text-decoration work in this case?

Comment (chiaberry) [reply code/signs-markings/signs-markings.css:85]: I mean a comment with `//` instead of `/* */`

File: code/signs-markings/signs-markings.js [hunk 2]
```diff
@@ -333,86 +333,94 @@ $(document).on("knack-view-render.view_2607", function(event, scene) {
...
-  div_id,
+/**
+ * Template and append a button link, disable it optionally, and invoke a callback function argument
+ * @parameter {string} id - id attribute of the a tag in the button link
+ * @parameter {string} view_id - Knack view id to append button link to
+ * @parameter {string} url - Destination to navigate to on click
+ * @parameter {string} fa_icon - Icon string (https://support.knack.com/hc/en-us/articles/226165208-Working-with-Icons#2-complete-list-of-icons)
+ * @parameter {bool} isDisabled - Is button disabled (defaults to false)
+ * @parameter {function} callback - Function that is invoked after appending the button link
+ */
```
Comment (chiaberry) [inline code/signs-markings/signs-markings.js:344]: <3

**Review by chiaberry — APPROVED** [review]

File: code/signs-markings/signs-markings.css (hunk 1 above)
Comment (mddilley) [reply code/signs-markings/signs-markings.css:85]: Oh nice catch! I'll update this. 🙏

**Pull request merged**